dktest.go

```diff
@@ -167,42 +168,57 @@ func waitContainerReady(ctx context.Context, lgr logger, c ContainerInfo,
 
 // Run runs the given test function once the specified Docker image is running in a container
 func Run(t *testing.T, imgName string, opts Options, testFunc func(*testing.T, ContainerInfo)) {
+	err := RunContext(context.Background(), t, imgName, opts, func(containerInfo ContainerInfo) error {
+		testFunc(t, containerInfo)
+		return nil
+	})
+	if err != nil {
+		t.Fatal("Failed to", err)
```

Comment:
The error messages below don't use "Failed to" since error strings are lowercase, and the wrapping can end up with multiple "failed to",
https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

However, keeping the "Failed to" in the `t.Fatal`, so that the end-user experience remains the same for almost all the cases.